package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@rdohms/gh-labeller",
+  "version": "0.1.0",
+  "description": "CLI Command that creates GitHub Labels from a set of configuration",
+  "main": "cli.js",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^4.19.1",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-prettier": "^2.6.2",
+    "jest": "^23.5.0",
+    "prettier": "^1.14.2"
+  },
+  "dependencies": {
+    "@octokit/rest": "^16",
+    "chalk": "^2.4.1",
+    "load-json-file": "^5.0.0",
+    "lodash": "^4.17.10",
+    "meow": "^5.0.0",
+    "proxyquire": "^2.1.0"
```

Comment:
Are we using `proxyrequire`? If not, we can remove it!

Reply:
aha! i was ahead of you there :P